components/scream/src/physics/p3/micro_p3_iso_f.f90

```diff
@@ -202,6 +202,33 @@ subroutine compute_rain_fall_velocity_f(qr_incld, rcldm, rhofacr, nr, nr_incld,
     real(kind=c_real), intent(out) :: mu_r, lamr, V_qr, V_nr
   end subroutine compute_rain_fall_velocity_f
 
+  subroutine ice_cldliq_collection_f(rho, t, rhofaci, f1pr04, qitot_incld, qc_incld, nitot_incld, &
+                                     nc_incld, qccol, nccol, qcshd, ncshdc) bind(C)
+    use iso_c_binding
+    
+    ! arguments:
+    real(kind=c_real), value, intent(in) :: rho, t, rhofaci, f1pr04
+    real(kind=c_real), intent(in) :: qitot_incld, qc_incld, nitot_incld, nc_incld
```

Comment:
Should this have the `value` annotation?

Reply:
Aha, This value key word is not necessary, it only requires in the micro_p3_iso_c.f90, I will take it out.